kas/jetson-agx-orin-devkit.yml

```diff
@@ -0,0 +1,11 @@
+header:
+  version: 14
+  includes:
+  - kas/include/mender-full.yml
+  - kas/include/tegra-jetpack6.yml
```

Comment:
I have only tested with JP5. I suspect you might be the first to try it with JP6.

Does it build if you rename the `bbappend` file? While I am not certain how to resolve the renamed recipe issue correctly, it would be interesting to hear how much more work remains to be done, as I eventually plan to give JP6 a go as well.

Reply:
It builds, but there are still some other fixes needed to work properly. Though most of them seem to be compatible with JP 6.
I'm still in the process of figuring out all required changes :D

Reply:
@apbr I'll go ahead and merge it as-is then, to reduce the amount of things which need to be carried along for JP6. To me its nicer having a state in-branch which already builds and "just" needs some more polish, instead of having to keep a whole separate state externally for even working on JP6.